### PR TITLE
Am/chore/doc casting

### PR DESCRIPTION
**Pull request by IceTDrinker**

### CODE REVIEW

File: tfhe/docs/guides/zk-pok.md [hunk 1]
```diff
@@ -69,3 +69,67 @@ Performance can be improved by setting `lto="fat"` in `Cargo.toml`
 lto = "fat"
 ```
 and by building the code for the native CPU architecture and in release mode, e.g. by calling `RUSTFLAGS="-C target-cpu=native" cargo run --release`.
+
+{% hint style="info" %}
+You can choose a more costly proof with `ZkComputeLoad::Proof` which has a faster verification time or the opposite with `ZkComputeLoad::Verify` for a faster proof and a slower verify.
```
Comment (yuxizama) [inline tfhe/docs/guides/zk-pok.md:74]:
```suggestion
You can choose a more costly proof with `ZkComputeLoad::Proof`, which has a faster verification time.  Alternatively, you can select `ZkComputeLoad::Verify` for a faster proof and slower verification.
{% endhint %}
```

File: tfhe/docs/guides/zk-pok.md [hunk 2]
```diff
@@ -69,3 +69,67 @@ Performance can be improved by setting `lto="fat"` in `Cargo.toml`
...
+{% endhint %}
+
+## Using dedicated Compact Public Key parameters
+
+**TFHE-rs** offers the ability to use dedicated parameters for the Compact Public Key encryption which can allow to reduce the size of encrypted data and speed-up the zero-knowledge proof computation.
```
Comment (yuxizama) [inline tfhe/docs/guides/zk-pok.md:79]:
```suggestion
You can use dedicated parameters for the Compact Public Key encryption to reduce the size of encrypted data and speed up the zero-knowledge proof computation.
```

File: tfhe/docs/guides/zk-pok.md [hunk 3]
```diff
@@ -69,3 +69,67 @@ Performance can be improved by setting `lto="fat"` in `Cargo.toml`
...
+
+This works essentially in the same way as before, indicating the dedicated parameters being the only additional step needed:
```
Comment (yuxizama) [inline tfhe/docs/guides/zk-pok.md:81]:
```suggestion
This works essentially in the same way as before. Additionally, you need to indicate the dedicated parameters:
```

**Review by yuxizama — COMMENTED** [review]
I've suggested some rewordings :)
Thanks!

**Review by jborfila — APPROVED** [review]

**Pull request merged**